build.gradle

```diff
@@ -0,0 +1,62 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        maven { url "https://oss.sonatype.org/content/groups/public"}
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'org.ajoberstar:gradle-git:1.6.0'
+    }
+}
+
+
+repositories {
+    mavenCentral()
+    maven {
+        url 'http://dnjnookrepo.barnesandnoble.com:8081/nexus/content/repositories/public'
+        artifactUrls 'http://dnjpanexus02.barnesandnoble.com/content/groups/public'
+    }
+    maven { url "http://dl.bintray.com/ajoberstar/maven" }
+}
+
+configurations {
+    jniLibs
+}
+
+apply plugin: 'org.ajoberstar.grgit'
+
+task addTag{
+    doLast{
+        grgit.tag.add{
+            name = version
+            message = "release of ${version}"
+        }    
+    }
+}
+
+task addJSs{
+    doLast{
+        grgit.add(patterns: ['build-output'])
+    }
+}
+
+task gitCommit(dependsOn: addJSs){
+    doLast{
+        grgit.commit{
+            message = "NO-JIRA: commit new generated JS files, ${version}"
+        }
+    }
+}
+
+task pushToRemote(dependsOn: gitCommit){
+    doLast{
+        grgit.push()
+    }
+}
+
+task pushTags(dependsOn: addTag){
+    doLast{
+        grgit.push(tags:true)
+    }
+}
+
```

Comment:
This seems very specific to `barnesandnoble.com`, and it is Android-related, right? (not generic `readium-shared-js` code)